source/retryable-writes/retryable-writes.rst

```diff
@@ -323,79 +315,50 @@ a writable server is reselected for the retry attempt.
 
 .. _Command Monitoring: ../command-monitoring/command-monitoring.rst
 
+Each attempt of a retryable write operation SHOULD report a different
+``requestId`` so that events for each attempt can be properly correlated with
+one another.
+
 The `Command Monitoring`_ specification states that the ``operationId`` field is
 a driver-generated, 64-bit integer and may be “used to link events together such
-as bulk write operations.” Drivers SHOULD NOT use this field to relay
-information about a transaction ID. A bulk write operation may consist of
-multiple write commands, each of which have a unique transaction ID.
-
-Drivers MUST add an optional ``transactionId`` field to the
-``CommandStartedEvent``, ``CommandSucceededEvent``, and
-``CommandFailedEventdata`` data structures:
-
-.. code:: typescript
-
-  /**
-   * Returns the transaction ID for the command. This is used to link
-   * events together such as retryable write operations. OPTIONAL.
-   *
-   * If set, this should be a subset of the command document containing
-   * only the sessionId and txnNum fields.
-   */
-  transactionId: Document;
-
-Drivers MUST set the ``transactionId`` field for write commands executed within
-a DriverSession where retryable writes have been enabled.
+as bulk write operations.” Each attempt of a retryable write operation SHOULD
```

Comment:
Please replace the smart quotes with straight quotes.

Reply:
Per our in-person chat, smart quotes are scattered throughout this repo and I was careful to only use them in prose (not code examples). Will leave as-is but a future PR can always clean them up across the repo.